docs/quickstarts/outlook-quickstart-json-manifest.md

```diff
@@ -108,6 +111,9 @@ The add-in project that you've created with the Yeoman generator contains sample
 
 1. View a message in the [Reading Pane](https://support.microsoft.com/office/2fd687ed-7fc4-4ae3-8eab-9f9b8c6d53f0), or open the message in its own window. A new control group named **Contoso Add-in** appears on the Outlook **Home** tab (or the **Message** tab if you opened the message in a new window). The group has a button named **Show Taskpane** and one named **Perform an action**.
 
+    > [!NOTE]
+    > If the new group isn't present, then your add-in wasn't automatically sideloaded. Follow the instructions in [Sideload manually - Outlook 2016 or later on Windows or Mac](../outlook/sideload-outlook-add-ins-for-testing.md#outlook-2016-or-later-on-windows-or-mac) to manually sideload the add-in in Outlook. When you are prompted to upload the manifest file, use the file `C:\Users\{your_user_name}\AppData\Local\Temp\manifest.xml`. The file has an `.xml` extension because during the preview period, the JSON-formatted manifest is converted to an XML manifest, which is then sideloaded.
```

Comment:
Just curious, did you ever have the first sideload attempt fail?

Reply:
I think so, but I'm not 100% sure.